.github/workflows/add-team-to-ghsa.yml

```diff
@@ -0,0 +1,22 @@
+name: Add Security Team to GHSAs
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  add-team-to-ghsa:
+    runs-on: macos-latest
```

Comment:
nit: does `ubuntu-latest` work for us? macos has a lower number of concurrent jobs

Reply:
Unfortunately no. I'm not sure why but the jq command breaks in ubuntu. Fought with it for a couple of hours before deciding macos was good enough 🤷

Reply:
is it related to the jq version? could you point me to the error message or the line? I'm happy to debug with it!

Reply:
Yeah it's [line 17 in the shell script](https://github.com/anza-xyz/agave/pull/2764/files#diff-81cff60a7ddfd44e6cd1f6cb98f629825b0183a542370397a223a2d722becb03R17)

This will reproduce. raw_ghsa.json has the contents of the gh call from lines 11-14:
`jq -r '[ .[] | select(all(.collaborating_teams.[]; .slug != "ghsa-testing-2"))| .ghsa_id ] | sort | .[] ' raw_ghsa.json`

Works on mac jq version v1.7 but not on ubuntu version 1.6

The error on ubuntu is:
```
jq: error: syntax error, unexpected '[', expecting FORMAT or QQSTRING_START (Unix shell quoting issues?) at <top-level>, line 1:
[ .[] | select(all(.collaborating_teams.[]; .slug != "ghsa-testing-2"))| .ghsa_id ] | sort | .[]
jq: 1 compile error
```

If I'm remembering correctly it comes from the `.[]` iterateor on collaborating_teams.

Reply:
wanna give `ubuntu-24.04` a try :trollface: it installs `jq 1.7` https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md

Reply:
Yeah that works! I thought I tried `ubuntu-latest` and it failed but maybe I didn't, or maybe `latest` didn't get `24.04` 🤔 

Anyway I just run it as a workflow with `runs-on: ubuntu-24.04` in my private repo and it worked fine. Thanks for figuring that out